src/analytics/private/sqls/redshift/event-parameter.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS {{schema}}.{{table_event_parameter}}(
+    event_timestamp BIGINT,
+    event_id VARCHAR(255) DEFAULT RANDOM(),
+    event_param_key VARCHAR(255),
+    event_param_double_value DOUBLE PRECISION,
+    event_param_float_value DOUBLE PRECISION,
+    event_param_int_value BIGINT,
+    event_param_string_value VARCHAR(255)    
+) DISTSTYLE EVEN 
+SORTKEY(event_id)
```

Comment:
id is a random string, how to use it for optimizing the query?

@qiaow02 @luorobin-a2z any suggestion?

Reply:
add event_name to this table, add changed the `SORTKEY(event_timestamp, event_name)`, make the SORTKEY same as `event` table, we can get better performance when do join with event.